templates/terraform/extra_schema_entry/sslCertificate.go.erb

```diff
@@ -0,0 +1,17 @@
+"name_prefix": {
```

Comment:
Thumbs up.  Problem - consider using name `ssl_certificate` instead of `sslCertificate`.

Reply:
Done